src/lib/components/mods/ModGrid.svelte

```diff
@@ -45,12 +45,15 @@
   let totalMods: number;
 
   let searchField = search;
+  $: showPagination = $mods && !$mods.error && !$mods.fetching;
```

Comment:
This hides the paginator when fetching the next page of mods after clicking a pagination button. I think the intent was to hide it only when an error occurs